puddlestuff/loadshortcuts.py

```diff
@@ -33,6 +33,7 @@ def check_file(path, resource):
         cparser = PuddleConfig(path)
         version = cparser.get('info', 'version', 0)
         if version < __version__:
+            print('Replacing version', version, 'config file', path, 'with newer version')
```

Comment:
i'd probably use a f-string here

should we make it clearer this refers to shorcuts?